src/Cards/CurrentCard.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import {
+  Card,
+  Question,
+  Option,
+  SubmitButton,
+  CardWindowBar,
+  ButtonContainer,
+} from "./styled";
+
+import close from "../images/close.svg";
+
+const CurrentCard = ({
+  question,
+  setScore,
+  score,
+  getNextQuestion,
+  isSubmit,
+  setIsSubmit,
+  isCorrect,
+  setIsCorrect,
+  isIncorrect,
+  setIsIncorrect,
+  questionNumber,
+}) => {
+  const [selectedOption, setSelectedOption] = useState("");
+  const [isEmpty, setIsEmpty] = useState(false);
+
+  function setSelectedValue(option) {
+    if (!isSubmit) {
+      setSelectedOption(option);
+    }
+  }
+  function checkSelect(selected) {
+    if (selected === selectedOption) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  function checkCorrectAnswer(option) {
```

Comment:
As pointed out below, the prop `isCorrect` in the Option component should receive a true or false. Therefore, this function should just return true or false. 
`const checkCorrectAnswer = (option) => option === question.correct`